guides/common/modules/ref_permissions-required-for-provisioning.adoc

```diff
@@ -0,0 +1,161 @@
+[id="permissions-required-for-provisioning_{context}"]
+= Permissions Required for Provisioning
+
+The following table lists the permissions a non-admin user requires to provision hosts.
+
+.Permissions required to provision hosts
+[options="header"]
+|====
+|Resource type|Permissions|Additional information
+
+|Compute resource
+|view_compute_resources
+
+view_compute_resources_vms
+
+create_compute_resources_vms
+
+destroy_compute_resources_vms
+
+power_compute_resources_vms
+|
+Required for provisioning of VM hosts
+
+Not required for bare-metal provisioning
+
+|Location
+|view_locations
+|
+
+|Subnet
+|view_subnets
+|
+
+|Hostgroup
+|view_hostgroups
+
+create_hostgroups
+
+edit_hostgroups
+
+play_roles_on_hostgroup
+|
+
+|Organization
+|view_organizations
+|
+
+|Compute profile
+|view_compute_profiles
+|
+
+|Host
+|view_hosts
+
+create_hosts
+
+edit_hosts
+
+destroy_hosts
+
+build_hosts
+
+power_hosts
+
+play_roles_on_host
+|
+
+|Host
+|
+view_discovered_hosts
+
+submit_discovered_hosts
+
+auto_provision_discovered_hosts
+
+provision_discovered_hosts
+
+edit_discovered_hosts
+
+destroy_discovered_hosts
+|Only in environments with the Foreman Discovery plug-in enabled
+
+|Activation Keys
+|view_activation_keys
+|
+
+|Domain
+|view_domains
+|
+
+|Environment
+|view_environments
+|
+
+|Image
+|view_images
+|
+
+|Medium
+|view_media
+|
+
+|Operatingsystem
+|view_operatingsystems
+|
+
+|Ptable
+|view_ptables
+|
+
+|Provisioning template
+|view_provisioning_templates
+|
+
+|Architecture
+|view_architectures
+|
+
+|Parameter
+|view_params
+
+create_params
+
+edit_params
+
+destroy_params
+|
+
+|Katello::KTEnvironment
+|view_lifecycle_environments
+|Only in environments with Katello
+
+|Katello::Product
+|view_products
+|Only in environments with Katello
+
+|Content View
+|view_content_views
+|Only in environments with Katello
+
+
+|Ansible role
+|view_ansible_roles
+|Only in environments that use Ansible
+
+|{SmartProxy}
+|view_smart_proxies
+
+view_smart_proxies_puppetca
+|
+
+|{SmartProxy}
+|view_openscap_proxies
+|Only in environments with openscap enabled
+|====
+
+[role="_additional-resources"]
```

Comment:
This is currently unused; we generally don't add anchors for "non-headings". Fine for me if you require it downstream. If not, please drop.

Reply:
It's part of the template generated by [newdoc](https://redhat-documentation.github.io/newdoc/). If it doesn't break anything upstream or for orcharhino, I'd prefer to keep it in. I'd rather not diverge from the modular documentation templates unless absolutely necessary.